hudi-common/src/main/avro/HoodieRestoreMetadata.avsc

```diff
@@ -34,6 +34,8 @@
         "name":"version",
         "type":["int", "null"],
         "default": 1
-     }
+     },
+     /* overlaps with 'instantsToRollback' field. Adding this to track action type for all the instants being rolled back. */
+     {"name": "restoreInstantInfo", "default": null, "type": {"type": "array", "default": null, "items": ["null", "HoodieInstantInfo"]}}
```

Comment:
Can you use pretty print mode (multi-lines) in the same way other section is presented.